src/app/examples/button-example/button-example.component.tns.html

```diff
@@ -1,9 +1,9 @@
 <StackLayout>
   <Button kirby-button text="Kirby Button"></Button>
-  <Button kirby-button text="Kirby Rounded Button" shape="round"></Button>        
-  <Button kirby-button text="Kirby Block Button" expand="block"></Button>        
-  <Button kirby-button text="Round Block Cake Buttons" expand="block" shape="round"></Button>       
+  <Button kirby-button text="Kirby Rounded Button" shape="round"></Button>
+  <Button kirby-button text="Kirby Block Button" expand="block"></Button>
+  <Button kirby-button text="Round Block Cake Buttons" expand="block" shape="round"></Button>
   <Button kirby-button text="Primary Button" shape="round" colortype="primary"></Button>
   <Button kirby-button text="Secondary Button" shape="round" colortype="secondary"></Button>
-</StackLayout>
-
+  <Button kirby-button text="Disabled Button" shape="round" isEnabled="false"></Button>
```

Comment:
We don't use `shape` anymore
```suggestion
  <Button kirby-button text="Disabled Button" isEnabled="false"></Button>
```

Reply:
Removed